apis/extension/slo_controller_config.go

```diff
@@ -35,15 +35,15 @@ const (
 // +k8s:deepcopy-gen=true
 type NodeCfgProfile struct {
 	// like ID for different nodeSelector; it's useful for console so that we can modify nodeCfg or nodeSelector by name
-	Name string `json:"name,omitempty"`
+	Name string `json:"name,omitempty" validate:"required"`
```

Comment:
This should be optional by default, and the check could be enabled via a command line argument.